modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/NodeImpl.java

```diff
@@ -3635,6 +3673,16 @@ public void changePeersAndLearners(final Configuration newPeersAndLearners, long
                     return;
             }
 
+            if (this.conf.getConf().getSequenceToken() > newPeersAndLearners.getSequenceToken()) {
+                 LOG.info("Node {} received stale configuration for conf {}, existing is {}, new {}.",
```

Comment:
According to our logging rules, we should put all params in [] at the end of the statement.
Besides I'd also mention that given configuration will be ignored.

Reply:
I used this block of code as an example:
```
  LOG.warn("Node {} ignored the configuration because of mismatching terms. Current term is {}, but provided is {}.",
                    getNodeId(), currentTerm, term);
```

Nevertheless, fixed

Reply:
Yep, original raft logs do not match our rules, but every second Monday of the month we rework one message.